### PR TITLE
fixed styled of the home page

### PR DESCRIPTION
**Pull request by ValentinCop**

### CODE REVIEW

**Review by ghost — DISMISSED** [review]
ok for me

File: frontend/src/pages/Home/style.jsx [hunk 1]
```diff
@@ -8,12 +8,15 @@ export default styled.div`
...
+    margin-top: 10vh;
+  }
+
   .logo {
-    height: 30vh;
     width: auto;
     text-align: center;
     display: block;
-    margin: auto;
+    margin: 2rem auto auto auto;
```
Comment (LuigiMenez) [inline frontend/src/pages/Home/style.jsx]: That's a lot of auto

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
That's a lot of auto

**Review by ghost — APPROVED** [review]
ok for me

**Review by LoicBrassart — APPROVED** [review]
SnE

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Pull request merged**